internal/handlers/pg/fetch.go

```diff
@@ -73,3 +73,37 @@ func nextRow(rows pgx.Rows) (*types.Document, error) {
 
 	return doc.(*types.Document), nil
 }
+
+func (h *Handler) protoFetch(ctx context.Context, param fetchParam) ([]*types.Document, error) {
```

Comment:
could you please share with us your thought on why the function name is like this?

Reply:
Hi, `protoFetch` is used in msg find and in tests and nowhere else.  
Why not rename `fetch` and add all code to this?

Reply:
actually I'm going to leave `fetch` func.  `protoFetch` is a temporarily prototype as a future view of `fetch` func. I'm not sure if you all okay with this new implementation so I just replaced this in one in one place (for simplicity) and waiting for you approval and command to apply it in every occurrences.